core/API/DataTableManipulator/LabelFilter.php

```diff
@@ -102,6 +106,9 @@ private function doFilterRecursiveDescend($labelParts, $dataTable)
     protected function manipulateSubtableRequest($request)
     {
         unset($request['label']);
+        unset($request['flat']);
+        $request['totals'] = 0;
+        $request['filter_sort_column'] = ''; // do not sort, we only want to find a matching column
```

Comment:
This makes label filter a bit faster as well since we will no longer sort the results which can be beneficial especially on large dataTables.